content/en/docs/languages/go/generated-code.md

```diff
@@ -3,12 +3,12 @@ title: Generated-code reference
 linkTitle: Generated code
 weight: 95
 ---
-
-This page describes the code generated with the [grpc plugin](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc), `protoc-gen-go-grpc`,
-when compiling `.proto` files with `protoc`.
+This page describes the code generated when compiling `.proto` files with `protoc`, using the `protoc-gen-go-grpc` [grpc plugin](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc). This latest version of generated code uses generics by default. If you're working with older generated code that doesn't use generics, you can find the relevant documentation [here](/docs/languages/go/generated-code-old).  While we encourage using this latest version with generics, you can temporarily revert to the old behavior by setting the `useGenericStreams` flag to `false`.
 
 You can find out how to define a gRPC service in a `.proto` file in [Service definition](/docs/what-is-grpc/core-concepts/#service-definition).
 
+To implement streaming in gRPC, refer directly to the [ServerStream](https://pkg.go.dev/google.golang.org/grpc#ServerStream) and [ClientStream](https://pkg.go.dev/google.golang.org/grpc#ClientStream) documentation, which provides comprehensive instructions for effectively using these types in your service methods.
```

Comment:
This links to the types that are embedded in the generic streaming types.  Better documentation for using the generic streaming types is present on the generic streaming types themselves.

This doc should call out the documentation on those generic streaming types.  The doc strings for the generic streaming types already reference ClientStream and ServerStream.